### DISCUSSION TITLE
css file is not getting minified with cssnano

### DISCUSSION THREAD

**Discussion by Arunjenson** [tailwindlabs/tailwindcss#14032] [General]
im trying to minify the css file for the production build using cssnano but even after adding the 
```
module.exports = {
  plugins: {
    tailwindcss: {},
    autoprefixer: {},
    cssnano: {
      preset: 'default',
    },
  },
};


```

there is no change in the file size

**Comment by franktucker907** [comment]
Hello,
Environment Variable: Make sure your NODE_ENV is set to production. cssnano will only minify the CSS if it detects that it’s in production mode.
NODE_ENV=production npm run build

Order of Plugins: Ensure that cssnano is the last plugin in your list. This ensures that it processes the CSS after all other transformations.
JavaScript
[Official Website](https://www.mycenturahealth.com.co)

module.exports = {
  plugins: {
    tailwindcss: {},
    autoprefixer: {},
    ...(process.env.NODE_ENV === 'production' ? { cssnano: { preset: 'default' } } : {}),
  },
};

Build Command: If you’re using a build tool like Webpack, ensure that your build command is correctly set up to use the production configuration.
Check Output: Sometimes, the minification might not be very noticeable if the original CSS is already optimized. You can compare the before and after sizes to see if there’s any reduction.
Additional Compression: Consider using additional compression techniques like Brotli or Gzip to further reduce the file size after minification.
Best Regards
franktucker907
